### ISSUE TITLE
[Gradio 4] Trimming breaks the video

### ISSUE THREAD

**Issue by hysts** [gradio-app/gradio#6203]
### Describe the bug

Trimming the beginning of the video corrupts the video. (Not 100% sure and it's weird, but looks like trimming from the end or from both side don't break the video.)

https://github.com/gradio-app/gradio/assets/25161192/3c7338fd-e36a-4f64-aeb0-a1c7e8fd9519

### Have you searched existing issues?  🔎

- [X] I have searched and found no existing issues

### Reproduction

```py
import gradio as gr

gr.Interface(fn=lambda x: x, inputs="video", outputs="video").launch()
```

### System Info

```shell
gradio==4.0.2
```

### Severity

I can work around it

**Comment by freddyaboulton** [comment]
I can't repro on the latest deployed version on spaces @hysts  [space](https://huggingface.co/spaces/gradio/video_component)

Do you mind trying again to see if the problem persists? If so please share the video


https://github.com/gradio-app/gradio/assets/41651716/40571bc6-50a7-4c79-8147-0d56b35250a4

**Comment by hysts** [comment]
@freddyaboulton Thanks for checking. But it still persists for me. I used the example video of the Space, but the same error occurred:

https://github.com/gradio-app/gradio/assets/25161192/c0a9cd0d-45ba-4ca9-9151-c20021c6177d

BTW, in your attached video, you are trimming the video from both sides. As I wrote in my comment, the error only occurs when the video is trimmed from the front only. Does it work for you when you trim the input from the front only?

> it's weird, but looks like trimming from the end or from both side don't break the video.
